namespace cpp entity
namespace py ccentity.gender
namespace as3 com.dreamfire.sanguo.data.definition.entity

enum GenderType {
  MIN = 0,

  MALE,               // 男性
  FAMAIL,             // 女性
  BOTH,               // 无关

  MAX,
}

